docs/containers.conf.5.md

```diff
@@ -867,6 +867,7 @@ default of 3.
 **podmansh_timeout**=30
```

Comment:
Just remove this section, We will use it if it set, but no reason to continue to document it.

Reply:
Done.